contracts/nft/contracts/zetachain/UniversalNFT.sol

```diff
@@ -1,13 +1,6 @@
 // SPDX-License-Identifier: MIT
```

Comment:
I would personally name the `zetachain` directory `zevm` to be consistent with protocol-contracts

Reply:
I'm all for consistency, but in my opinion zEVM is confusing. I'm using `zetachain` instead of `zevm` intentionally to make it easier for developers to understand, which chain a contract is meant to be deployed on. `zevm` is somewhat ok inside the protocol contracts as an internal shorthand, but for a wider audience, I don't think it's suitable.